charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -473,6 +473,10 @@ collectors:
             http:
               endpoint: 0.0.0.0:4318
       processors:
+        batch:
+          send_batch_size: 1000
+          timeout: 1s
+          send_batch_max_size: 1500
```

Comment:
Can we rely on the processor's default values and change this to `batch: {}`?

Reply:
I try to be explicit where possible, IMO the default batch size being 8k is too large